### ISSUE TITLE
BUG: Drag & drop indicators are not shown in inline editor

### ISSUE THREAD

**Issue by Sebobo** [neos/neos-ui#3977]
CKEditor supports showing indicators when dragging and dropping selections, but the Neos UI is missing the styles for them, even though the features is installed and enabled.

Can be seen when playing with the editor on
https://ckeditor.com/ckeditor-5/demo/feature-rich/
